corehq/apps/enterprise/tasks.py

```diff
@@ -36,7 +36,7 @@ def email_enterprise_report(domain, slug, couch_user):
     # Send email
     url = absolute_reverse("enterprise_dashboard_download", args=[domain, report.slug, str(hash_id)])
     link = "<a href='{}'>{}</a>".format(url, url)
-    subject = _("Enterprise Dashboard: {}").format(report.title)
+    subject = _("Enterprise Console: {}").format(report.title)
```

Comment:
please revert this change. should remain enterprise dashboard as we are just changing the section name